client/src/components/graph/graph.tsx

```diff
@@ -1077,6 +1145,18 @@ class Graph extends React.Component<GraphProps, GraphState> {
         data-camera-distance={camera?.distance()}
         ref={this.graphRef}
       >
+        <button
```

Comment:
This button style and behavior will be addressed in this ticket:

https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-explorer/910